test/cypress/support/commands.js

```diff
@@ -37,13 +40,50 @@ Cypress.Commands.add('giLogin', (userName, password = '123456789') => {
   cy.getByDT(elName).should('contain', userName)
 })
 
+Cypress.Commands.add('giLogOutWithNoGroup', () => {
+  cy.getByDT('logout').click()
+  cy.getByDT('welcomeHome').should('contain', 'Welcome to GroupIncome')
+})
+
 Cypress.Commands.add('giLogOut', () => {
   cy.getByDT('settingsBtn').click()
   cy.getByDT('link-logout').click()
 
   cy.getByDT('welcomeHome').should('contain', 'Welcome to GroupIncome')
 })
 
+Cypress.Commands.add('giCreateGroup', (groupName, groupImage, testValues, testIncome) => {
```

Comment:
Usually, when a function starts to get a lot of parameters (like creating a group), I find it more readable and scalable to pass a single object instead of N different parameters. `{ name, image, values, income }`. In the future we'll have to test the settings as well, so instead of adding more parameters, we just complement the object. 

Besides that, most of the stuff can be optional like we did with signUp.

```
Cypress.Commands.add('giCreateGroup', (groupName, {
 image: 'imageTest.png',
 purpose: 'Do great things!'
 income: 500,
} {
  // the command test...
}
```

when calling it, we just need to pass a proper name to it and if we want to customize one of the steps, it's easier.

```
// A group with 450$ income instead of the default 500$.
cy.giCreateGroup('Donnuts', { income: 450 });
```

Does this make sense to you?